spec/furniture/marketplace/stripe_events_controller_request_spec.rb

```diff
@@ -81,6 +97,22 @@
         expect(Stripe::Transfer).not_to(have_received(:create))
       end
     end
+
+    context "when Square notifications are not enabled" do
+      it "does not attempt to transfer the order to seller's Square dashboard" do
+        call
+        expect(square_client).not_to(have_received(:orders))
+      end
+    end
+
+    context "when Square notifications are enabled" do
+      let(:order) { order_for_square }
+
+      it "attempts to transfer the order to seller's Square dashboard" do
+        call
+        expect(square_client).to have_received(:orders)
```

Comment:
I tend to prefer more explicit expectations; as it stands we could just call `orders` and the tests would continue to pass. If we include the actual data structure it would be a bit more tightly coupled; but would also fail in the event we accidentally start munging about the data.

Once we start adding more permutations, we'll want to push the test down below the request spec level; and probably add a `Marketplace::Square::Order` class to encapsulate that logic.

Reply:
Yeah, I agree. I didn't want to get toooo detailed until I got some feedback first from y'all. I'll update with a `.with(...)` clause.